.gitignore

```diff
@@ -0,0 +1,41 @@
+### macOS ###
+*.DS_Store
+.AppleDouble
```

Comment:
Careful with using other's code within your project. I get this is a template .gitignore file, but there are items in here that unless you fully understand why they are in here, they should not be in *your* code base.